.github/workflows/build.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build
+on: [push]
+
+jobs:
```

Comment:
How about dividing the steps into more Jobs?
  * build
  * eslint
  * test (here at the moment it's also necessary to build)

Reply:
Can do